### PR TITLE
Add detail about customization, tree sharing, and decision point scope

### LINKED ISSUE #239: Add description of Decision Point Scope and Decision Tree Scope to Customization Guidance
See Discussion #223 for background detail.

- [x] Add a subsection to [/doc/md_src_files/060_decision-trees.md](https://github.com/CERTCC/SSVC/blob/eb94a2b50ec2eb26067ebccb86bf9b598b4d68e4/doc/md_src_files/060_decision-trees.md?plain=1#L88) under _Tree Construction and Customization Guidance_ to explain how customization might be shared among stakeholder communities 
   - touch on both the _Decision Point Scope_ topic as well as the _Decision Tree Scope_ as outlined in [this comment](https://github.com/CERTCC/SSVC/discussions/223#discussioncomment-5951133) in #223.

### PR DESCRIPTION
**Pull request by ahouseholder**
Resolves #239
Resolves #223

### CODE REVIEW

File: doc/md_src_files/060_decision-trees.md [hunk 1]
```diff
@@ -174,9 +187,53 @@ Thus, 60 unique combinations of decision values is the point at which a decision
...
+Each decision point defined in this document has a characteristic scope, either *global* or *local*.
+
+- *Globally scoped decision points* describe the state of the world outside the decision maker's environment. 
+They form the background context in which the stakeholder is making prioritization decisions. 
+Nearly all stakeholders should agree on the assignment of specific values to these decision points.
+- *Locally scoped decision points* are expected to be contextual to some decision makers.
+Mission Impact is one such example.
+Information about a locally scoped decision point can still be inherited by others using the same tree. 
+For example in the corporate CSIRT scenario above, the System Exposure value might be consistent across all subsidiaries for a centrally managed service.
+
```
Comment (ahouseholder) [inline doc/md_src_files/060_decision-trees.md:214]: For example, regulatory or uptime impacts

Use terminology like locally -> stakeholder-specific, global -> "objective"?

Comment (ahouseholder) [reply doc/md_src_files/060_decision-trees.md]: This comment has been addressed as of e223d7e803df2471afb2e700b7fe7a8b4d77d287

**Review by j--- — APPROVED** [review]
This is really good. I don't have any suggestions for improvement, it's well organized, concisely written, and flows well.

**Review by cgyarbrough — APPROVED** [review]
I approve the changes as submitted. We have discussed these previously.

**Pull request merged**